sys/net/gnrc/network_layer/ndp/gnrc_ndp.c

```diff
@@ -57,13 +57,17 @@ static void _stale_nc(kernel_pid_t iface, ipv6_addr_t *ipaddr, uint8_t *l2addr,
 #ifdef MODULE_GNRC_SIXLOWPAN_ND_ROUTER
             /* tentative type see https://tools.ietf.org/html/rfc6775#section-6.3 */
             gnrc_ipv6_netif_t *ipv6_iface = gnrc_ipv6_netif_get(iface);
-            if ((ipv6_iface == NULL) || (ipv6_iface->flags & GNRC_IPV6_NETIF_FLAGS_ROUTER)) {
+            if (ipv6_iface->flags &
+                (GNRC_IPV6_NETIF_FLAGS_SIXLOWPAN | GNRC_IPV6_NETIF_FLAGS_ROUTER)) {
```

Comment:
The statement is now `if (iface_is_router || iface_is_sixlowpan)`. Was that your intention?